Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/LandscapeIntegrationTest.java

```diff
@@ -99,6 +109,8 @@ public void testLandscapeMode() {
 
 	@Test
 	public void testTools() {
+		CountingIdlingResource idlingResource = activityTestRule.getActivity().getIdlingResource();
+		IdlingRegistry.getInstance().register(idlingResource);
```

Comment:
You already invoked .register() in the setup method, so I think there is no need to additionally .register() at the start of the test.